profiles/common/modules/features/nexteuropa_webtools/nexteuropa_webtools.module

```diff
@@ -21,6 +28,14 @@ function nexteuropa_webtools_menu() {
     'file' => 'nexteuropa_webtools.admin.inc',
     'type' => MENU_NORMAL_ITEM,
   );
+
+  $items['nexteuropa/webtools-ckeditor/%'] = array(
+    'page callback' => 'nexteuropa_webtools_ajax_callback',
+    'page arguments' => array(2, 3),
```

Comment:
nexteuropa_webtools_ajax_callback() only accepts one argument, probably 3 is not applicable here?